llama_index/llama_dataset/base.py

```diff
@@ -169,25 +174,46 @@ def _predict_example(
         """
 
     def make_predictions_with(
-        self, query_engine: BaseQueryEngine, show_progress: bool = False
+        self,
+        query_engine: BaseQueryEngine,
+        show_progress: bool = False,
+        batch_size: int = 20,
+        sleep_time_in_seconds: int = 10,
```

Comment:
Is 10s a little large? Or did you calculate this somehow using the default batch size?

Reply:
yeah i've been testing (painfully) using my personal openai api key which is `Usage Tier 1` just the next tier above Free, which is severely rate-limited.

`batch_size = 5` and `sleep_time_in_seconds = 15` actually is what seems to work without hitting RateLimitError for evaluator pack for Usage Tier 1

Reply:
i can set it to lower tho and for evaluator pack i'll make mention of these settings for usage tier 1

Reply:
Awesome, thanks for (painfully) finding the best settings